middle.mkd

```diff
@@ -35,12 +35,14 @@ to add or delete multiple records with one resource record (RR). MIXFR is
 DNSSEC aware thus if there is a change to RRset it knows to delete the
 covering RRISG(s), this saves the transmission of old RRsigs. 
 
-## Implicit RRSIG deletion
+## Implicit DNSSEC deletions
 
 When an RRset is modified, the MIXFR client MUST also
 remove all existing RRSIG records on that RRset. This is valid for all
 RRtypes except RRSIG itself.
 
+When an NSEC3PARAM is deleted or replaced, the MIXFR client MUST also remove all existing NSEC3 records on the zone that form the chain related to the deleted or replaced salt.
+
```

Comment:
I think "or replaced" is not needed, since replaced means one or more NSEC3PARAM deleted and one more more added. So replaced implicitly means one is deleted.

Reply:
To be more precise I think this should be:
```
When an NSEC3PARAM is deleted, the MIXFR client MUST also remove
all existing NSEC3 records on the zone with the same Hash Algorithm,
Flags, Iterations, and Salt.
```

Reply:
Also I think we should add some text here about the MIXFR server:
```
A MIXFR server MUST NOT transfer a delete of an NSEC3PARAM record unless it intends to 
remove the complete corresponding NSEC3 chain.
```